misc/sampleconfigs/i2c-lcd.service.default.sample

```diff
@@ -0,0 +1,13 @@
+[Unit]  
+Description=LCD Matrix Display Service
+After=network.target phoniebox-rfid-reader.service
+
+[Service]
+User=pi
+Group=pi
+Restart=always
+WorkingDirectory=/home/pi/RPi-Jukebox-RFID
+ExecStart=/usr/bin/python /home/pi/RPi-Jukebox-RFID/scripts/i2c_lcd.py
```

Comment:
Please remove the path to the executable here and add a shebang for Python3 in the Python file

Reply:
The comment is reflected in and resolved by the latest commit (*I hope*).